src/snap-points.ts

```diff
@@ -85,10 +86,8 @@ export const createSnapshotForNetwork = async (network: Network) => {
   );
 
   const finalLogs = txLogs.flat();
-  const initialEvents = JSON.parse(
-    fs.readFileSync(eventsSnapFilename, "utf-8")
-  );
-  const events = extractEvents(initialEvents, market, finalLogs);
+  const previousData = JSON.parse(fs.readFileSync(eventsSnapFilename, "utf-8"));
+  const events = extractEvents(convertJson(previousData), market, finalLogs);
```

Comment:
Why do you convert whole file? Most of them will not be used, you should convert it only when used.